### ISSUE TITLE
build.sh additional-namespaces.jsonnet throws an RUNTIME ERROR: couldn't open import "kube-prometheus/main.libsonnet": no match locally or in the Jsonnet library path

### ISSUE THREAD

**Issue by Epimetheus84** [prometheus-operator/kube-prometheus#1289]
**What happened?**
I've tried to install kube-prometheus and customized it to monitor several namespaces, but after executing _build.sh additional-namespaces.jsonnet_  I've got RUNTIME ERROR: couldn't open import "kube-prometheus/main.libsonnet": no match locally or in the Jsonnet library path.

**Did you expect to see some different?**

Generated yml files that I can apply and set prometheus to monitor several namespaces.

**How to reproduce it (as minimally and precisely as possible)**:

I've followed installation guide https://github.com/prometheus-operator/kube-prometheus#customizing-kube-prometheus

```
jb init
jb install github.com/prometheus-operator/kube-prometheus/jsonnet/kube-prometheus@release-0.8 # 0.7 throws same error
wget https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/release-0.8/build.sh -O build.sh

```
Then I've created .jsonnet file with configs by example https://github.com/prometheus-operator/kube-prometheus/blob/main/examples/additional-namespaces.jsonnet (just namespace names has been changed)

```
local kp = (import 'kube-prometheus/main.libsonnet') + {
  values+:: {
    common+: {
      namespace: 'monitoring',
    },

    prometheus+: {
      namespaces+: ['my-namespace', 'my-second-namespace'],
    },
  },
};

{ ['00namespace-' + name]: kp.kubePrometheus[name] for name in std.objectFields(kp.kubePrometheus) } +
{ ['0prometheus-operator-' + name]: kp.prometheusOperator[name] for name in std.objectFields(kp.prometheusOperator) } +
{ ['node-exporter-' + name]: kp.nodeExporter[name] for name in std.objectFields(kp.nodeExporter) } +
{ ['kube-state-metrics-' + name]: kp.kubeStateMetrics[name] for name in std.objectFields(kp.kubeStateMetrics) } +
{ ['alertmanager-' + name]: kp.alertmanager[name] for name in std.objectFields(kp.alertmanager) } +
{ ['prometheus-' + name]: kp.prometheus[name] for name in std.objectFields(kp.prometheus) } +
{ ['grafana-' + name]: kp.grafana[name] for name in std.objectFields(kp.grafana) }
```

But in the step "Compiling" `build.sh additional-namespaces.jsonnet` thrown an error:

```
+ set -o pipefail
++ pwd
+ PATH='/c/Users/eliseev/jb-folder/monitoring-stack/tmp/bin:/c/Users/eliseev/bin:/mingw64/bin:/usr/local/bin:/usr/bin:/bin:/mingw64/bin:/usr/bin:/c/Users/eliseev/bin:/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common:/c/Program Files/Python39/Scripts:/c/Program Files/Python39:/c/Program Files (x86)/Common Files/Oracle/Java/javapath:/c/WINDOWS/system32:/c/WINDOWS:/c/WINDOWS/System32/Wbem:/c/WINDOWS/System32/WindowsPowerShell/v1.0:/c/WINDOWS/System32/OpenSSH:/c/Program Files/Java/jdk1.8.0_281/bin:/c/Progra~1/maven/apache-maven-3.6.3/bin:/c/Program Files (x86)/Bitvise SSH Client:/cmd:/c/ProgramData/chocolatey/bin:/c/Program Files/PuTTY:/c/Program Files/Go/bin:/c/Python34/Scripts:/c/Users/eliseev/go/bin:/usr/bin/vendor_perl:/usr/bin/core_perl'
+ rm -rf manifests
+ mkdir -p manifests/setup
+ jsonnet -J vendor -m manifests example.jsonnet
+ xargs '-I{}' sh -c 'cat {} | gojsontoyaml > {}.yaml' -- '{}'
RUNTIME ERROR: couldn't open import "kube-prometheus/main.libsonnet": no match locally or in the Jsonnet library paths
        example.jsonnet:1:13-52 thunk <kp> from <$>
        example.jsonnet:13:81-83
        <std>:1293:24-25        thunk from <function <anonymous>>
        <std>:1293:5-33 function <anonymous>
        example.jsonnet:13:64-99        $
        example.jsonnet:13:1-101
        example.jsonnet:13:1-101
        During evaluation
```

* Kubernetes version information:

Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.1", GitCommit:"5e58841cce77d4bc13713ad2b91fa0d961e69192", GitTreeState:"clean", BuildDate:"2021-05-12T14:18:45Z", GoVersion:"go1.16.4", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.1", GitCommit:"5e58841cce77d4bc13713ad2b91fa0d961e69192", GitTreeState:"clean", BuildDate:"2021-05-12T14:12:29Z", GoVersion:"go1.16.4", Compiler:"gc", Platform:"linux/amd64"}

**Anything else we need to know?**:
jsonnet --version:
Jsonnet commandline interpreter v0.17.0

jb --version:
dev

**Comment by dgrisonnet** [comment]
I tried to reproduce the issue by following the step you provided and everything went fine. Could you perhaps try again from scratch?

If the problem persist, could you please verify that:
- `vendor/kube-prometheus/main.libsonnet` exists
- `jsonnetfile.json` points to the release of kube-prometheus you installed

**Comment by Epimetheus84** [comment]
It seems that cause in jb on windows. 
after `jb install github.com/prometheus-operator/kube-prometheus/jsonnet/kube-prometheus@release-0.8` has been completed 
vendor look like this

```
drwxr-xr-x 1 eliseev 1049089  0 Jul 29 15:19 ./
drwxr-xr-x 1 eliseev 1049089  0 Jul 29 15:33 ../
lrwxrwxrwx 1 eliseev 1049089 57 Jul 29 15:19 alertmanager -> github.com/prometheus/alertmanager/doc/alertmanager-mixin
drwxr-xr-x 1 eliseev 1049089  0 Jul 29 15:19 github.com/
lrwxrwxrwx 1 eliseev 1049089 44 Jul 29 15:19 grafana -> github.com/brancz/kubernetes-grafana/grafana
lrwxrwxrwx 1 eliseev 1049089 47 Jul 29 15:19 grafana-builder -> github.com/grafana/jsonnet-libs/grafana-builder
lrwxrwxrwx 1 eliseev 1049089 42 Jul 29 15:19 grafonnet -> github.com/grafana/grafonnet-lib/grafonnet
lrwxrwxrwx 1 eliseev 1049089 30 Jul 29 15:19 ksonnet -> github.com/ksonnet/ksonnet-lib
lrwxrwxrwx 1 eliseev 1049089 70 Jul 29 15:19 kube-prometheus -> github.com/prometheus-operator/kube-prometheus/jsonnet/kube-prometheus
lrwxrwxrwx 1 eliseev 1049089 67 Jul 29 15:19 kube-state-metrics -> github.com/kubernetes/kube-state-metrics/jsonnet/kube-state-metrics
lrwxrwxrwx 1 eliseev 1049089 73 Jul 29 15:19 kube-state-metrics-mixin -> github.com/kubernetes/kube-state-metrics/jsonnet/kube-state-metrics-mixin
lrwxrwxrwx 1 eliseev 1049089 49 Jul 29 15:19 kubernetes-mixin -> github.com/kubernetes-monitoring/kubernetes-mixin
lrwxrwxrwx 1 eliseev 1049089 37 Jul 29 15:19 mixin -> github.com/etcd-io/etcd/contrib/mixin
lrwxrwxrwx 1 eliseev 1049089 51 Jul 29 15:19 node-mixin -> github.com/prometheus/node_exporter/docs/node-mixin
lrwxrwxrwx 1 eliseev 1049089 63 Jul 29 15:19 prometheus -> github.com/prometheus/prometheus/documentation/prometheus-mixin
lrwxrwxrwx 1 eliseev 1049089 78 Jul 29 15:19 prometheus-operator -> github.com/prometheus-operator/prometheus-operator/jsonnet/prometheus-operator
lrwxrwxrwx 1 eliseev 1049089 64 Jul 29 15:19 prometheus-operator-mixin -> github.com/prometheus-operator/prometheus-operator/jsonnet/mixin
lrwxrwxrwx 1 eliseev 1049089 67 Jul 29 15:19 promgrafonnet -> github.com/kubernetes-monitoring/kubernetes-mixin/lib/promgrafonnet
lrwxrwxrwx 1 eliseev 1049089 33 Jul 29 15:19 thanos-mixin -> github.com/thanos-io/thanos/mixin
```

```
ls github.com shows that github.com folder is empty, symlinks link to nothing.
```

downgrading version of jb to 0.2.0 helps to fix this error, but, just for @release-0.4 version, installation of newer versions cause  `jb.exe: error: failed to install packages: downloading: failed to create tmp dir: CreateFile vendor\.tmp: The system cannot find the path specified..`

I've tried to repeat all steps in 2 different windows systems, same error occurs, but on ubuntu it works as expected with last jb version
![594f4c55-d923-41ac-90fc-564bdbecc6e9](https://user-images.githubusercontent.com/23120621/127534399-eaf14b9c-e44b-4104-bbc6-24004d16fbdd.jpg)

May be it is related with such jb install output, which says that folders in github.com are cleaned
![ef04bc90-3272-465b-ad06-b3e690cb0c05](https://user-images.githubusercontent.com/23120621/127534444-661be103-8bdf-4756-893f-a98435f6ac42.jpg)
